### ISSUE TITLE
title="" on cue points should remove HTML tags

### ISSUE THREAD

**Issue by dascritch** [dascritch/cpu-audio#178]
Any CPU show is plagued with `<i>` for any musical excerpt.

Culprit is in element_cpu.js `drawPoint()` function,
```js
                        elementPointTrack.title = text;
```
should be escaped with an inverse function of `escapeHtml()`  (in utils.js)
